src/sentry/apidocs/parameters.py

```diff
@@ -70,6 +70,24 @@ class GLOBAL_PARAMS:
         description="The name of environments to filter by.",
     )
 
+    def NAME(description: str, required: str = False) -> str:
+        return OpenApiParameter(
+            name="name",
+            location="query",
+            required=required,
+            type=str,
+            description=description,
+        )
+
+    def SLUG(description: str, required: str = False) -> str:
+        return OpenApiParameter(
+            name="slug",
+            location="query",
+            required=required,
+            type=str,
+            description=description,
+        )
```

Comment:
These should both be global with the option to set the description and required b/c we use name and slug across several API tabs (eg: project, org, release, teams) w/ diff reqs and descriptions.